app/src/main/java/com/ikarabulut/Rules2.java

```diff
@@ -0,0 +1,23 @@
+package com.ikarabulut;
+
+public class Rules2 implements RuleSet {
```

Comment:
May I please suggest the following names?

* DefaultRules
* StayAliveWithFour

Naming them after the player selection number is confusing, and will need to change if the options are ever reordered.

Reply:
Much more concise, thank you.

I was thinking DeadToAliveWith4Alive but felt that was too long

Reply:
👍 🎉

Reply:
Don't forget to change the test names and descriptions.  I mention that because it's easy to forget. I've definitely never done that. :)